docs/architecture/notifications.md

```
@@ -167,74 +170,86 @@ The gateway manager registers each as a separate adapter. Subscriptions referenc
-| Datadog | Webhooks | Stub |
-| Grafana | Webhooks | Stub |
-| Stripe | Webhooks | Stub |
-| Vercel | Webhooks | Stub |
-| Netlify | Webhooks | Stub |
-| Notion | Polling | Stub |
-| Twitch | IRC + EventSub | Stub |
-| Home Assistant | WebSocket + REST | Stub |
-| Nostr | NIP-04 DMs | Stub |
-| iMessage | BlueBubbles | Stub |
+### Tier 1: Chat Platforms
+
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)